lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -100,9 +106,15 @@ private void init(@Nullable AttributeSet attrs) {
         R.styleable.LottieAnimationView_lottie_cacheStrategy,
         CacheStrategy.Weak.ordinal());
     defaultCacheStrategy = CacheStrategy.values()[cacheStrategy];
-    String fileName = ta.getString(R.styleable.LottieAnimationView_lottie_fileName);
-    if (!isInEditMode() && fileName != null) {
-      setAnimation(fileName);
+    if (!isInEditMode()) {
+      String fileName = ta.getString(R.styleable.LottieAnimationView_lottie_fileName);
+      if (fileName != null) {
+        setAnimation(fileName);
+      }
+      int rawResId = ta.getResourceId(R.styleable.LottieAnimationView_lottie_src, 0);
```

Comment:
Hmm, instead of immediately overwriting the asset if res is also set, how about something like:
```
String assetName = ...
@ResId int rawResId = ...
if (assetName != null && rawResId != 0) {
    throw new IllegalArgumentException("...");
} else if (assetName != null) {
    ...
} else if (rawResId != 0) {
    ...
}
```

Reply:
I agree.

Reply:
But in your case `lottie_fileName` takes precedence. Is that something wanted?

Regarding the Exception, I would prefer sticking to only picking the preferred one (just like `android:padding` vs `android:paddingRight`). Could you please explain why you would go for an `Exception`?

I thought about an option that would simplify the use of the XML attribute but I didn't go into that path because it's really breaking the current API and the apps out there. The idea would be to get rid to `lottie_fileName` and replace it by `lottie_src` that could take either a `String` or a resource id to a raw file. Here is a quick list of the pros & cons I found:

Pros:
  - API similar to `ImageView`
  - Simpler API and no weird case with both set
 
Cons:
  - Break the current API

Reply:
@cyrilmottier Hmm, we could go with src and deprecate fileName and remove it in a future release.

I prefer failing loudly when you do unexpected behavior like set two conflicting APIs.